cpp/simulations/data_generation.sh

```diff
@@ -0,0 +1,23 @@
+#! /bin/bash
+# Run all scripts in memilio-epidata which are necessary to run the simulation files.
+
+# activate virutal_env 
+# The path to the virtual environment has to be adjusted individually
+path_virtual_env="$HOME/Documents/virtualenv/bin/activate"
```

Comment:
This only works on Linux, right?

Reply:
Only the path has to be adjusted. Otherwise, the script runs without an error on windows and linux.